src/nccl_ofi_rdma.c

```diff
@@ -1433,7 +1433,7 @@ static inline int process_completions(struct fi_cq_data_entry *cq_entry, uint64_
 
 			if (req->type == NCCL_OFI_RDMA_SEND_CONN || req->type == NCCL_OFI_RDMA_SEND_CONN_RESP) {
 				/* CONN or CONN_RESP send completion */
-				ret = inc_req_completion(req, cq_entry[comp_idx].len, 1);
+				ret = inc_req_completion(req, sizeof(nccl_ofi_rdma_connection_info_t), 1);
```

Comment:
Could be that I am not reading this right, but independent of this change - why would we increment the size of the request that came in from NCCL to account for the conn/conn_resp messages?

Reply:
This is an internal request that is not returned to NCCL. So the only place we actually use this size value is for tracing.

Reply:
Sounds good.